beachball.config.js

```diff
@@ -6,9 +6,10 @@
  */
 
 const {execSync} = require('child_process');
+const {findRepoPackageSync} = require('@react-native-windows/package-utils');
```

Comment:
This adds an unfortunate tradeoff. I previously added an optimization to only install beachball, and to skip building the repo. This doesn't allow us to use repo libraries though. E.g. `package-utils` has logic to correctly find packages in the repo, regardless of whether they are hoisted.

With this change, we `yarn install` the full repo, and build a very small subset of packages. This adds ~1 minute to the PR critical path, but allows us to have richer hooks into beachball.